website/content/docs/agent/caching/index.mdx

```diff
@@ -57,6 +57,30 @@ overrides the default behavior described above in [Using Auth-Auth
 Token](/docs/agent/caching#using-auto-auth-token), and instead ignores any
 existing Vault token in the request and instead uses the auto-auth token.
 
+## Persistent Cache
+
+Vault Agent restoring tokens and leases from a persistent cache 
```

Comment:
Should this be something like "restores" or "can restore"?

Reply:
Whoops, I had a change local which I never pushed. Thanks!